### DISCUSSION TITLE
Is there a way to right click a YT video?

### DISCUSSION THREAD

**Discussion by ghost** [qutebrowser/qutebrowser#7610] [Q&A]
I want to be able to open a youtube video's context menu using the keyboard. Is this accomplishable?

**Comment by mschilli87** [comment]
@boohba: Not sure what you are tryinbg to do and not willing to trouble-shoot Google's services but this sounds to me like you are looking for passthrough mode (Ctrl+V by default).

**Reply by ghost** [reply]
I'm trying to open this menu and click the Loop button.

![image_2023-03-13_09-15-11](https://user-images.githubusercontent.com/58115165/224712713-72a8b5d3-cc0a-40ce-be8b-265ee36486de.png)

**Reply by ghost** [reply]
Ctrl+V doesn't seem to do anything.

**Comment by benjaminbhollon** [comment]
I set up a keybinding to right click on a hinted element:

:bind ;; hint all right-click

(I think that's the right syntax; it's been a while.)

Then pressing ;; will bring up hints, the same as pressing f would, but the selected element is right clicked. I just tested, and it works to right-click on a youtube video.
